src/main/webapp/app/course/course-scores/scores-storage.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable } from '@angular/core';
+import { ScoresPerExerciseType } from 'app/entities/exercise.model';
+import { Result } from 'app/entities/result.model';
+import { CourseScores } from 'app/course/course-scores/course-scores';
+
+@Injectable({ providedIn: 'root' })
+export class ScoresStorageService {
+    private storedTotalScores: Map<number, CourseScores> = new Map();
```

Comment:
please document for all occurrences here, what the number represents, otherwise this is hard to understand. It is probably some kind of id, but we need to the the object type (e.g. exercise)